mlir/include/mlir/IR/AffineExprVisitor.h

```diff
@@ -413,7 +413,8 @@ class SimpleAffineExprFlattener
   /// lhs of the mod, floordiv, ceildiv or mul expression and with respect to a
   /// symbolic rhs expression. `localExpr` is the simplified tree expression
   /// (AffineExpr) corresponding to the quantifier.
-  virtual void addLocalIdSemiAffine(AffineExpr localExpr);
+  virtual void addLocalIdSemiAffine(AffineExpr localExpr, ArrayRef<int64_t> lhs,
```

Comment:
nit: I would prefer if these two APIs looked more consistent.  Could you give the members the same names and same orders? Also, please update docs for addLocalIdSemiAffine.